mate-base/mate-applets/mate-applets-1.26.0.ebuild

```diff
@@ -39,9 +39,7 @@ COMMON_DEPEND="
 		sys-power/cpupower
 		policykit? ( >=sys-auth/polkit-0.97:0 )
 	)
-	netlink? (
-		dev-libs/libnl:3
-	)
+	dev-libs/libnl:3
```

Comment:
is netlink no longer a configuration variable?

Reply:
If so this needs to be toggeled together with removing the USE flag at the top.

Reply:
I see your use flags:
```
* USE:        X abi_x86_64 amd64 cpupower elibc_glibc ipv6 kernel_linux policykit upower userland_GNU
```

Reply:
OK< I added netlink config option. Please check now.

Reply:
Looks. good